src/app/components/top-sellers/top-sellers.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="title-container">
+  <h1 class="title">Más vendidos</h1>
+</div>
```

Comment:
Why do you need a container for a header?

Reply:
To easily center text